risc0/zkvm/src/exec/env.rs

```diff
@@ -151,6 +213,39 @@ impl<'a> ExecutorEnvBuilder<'a> {
     }
 
     /// Add initial input that can be read by the guest from stdin.
+    /// This step must occur before the executor environment is built.
+    /// # Examples
+    /// ```
+    /// # use risc0_zkvm::{
+    /// #   ExecutorEnv,
+    /// #   ExecutorEnvBuilder,
+    ///     serde::to_vec};
```

Comment:
This `use` is dangling. Probably just un-hide the entire `use` block, but if you really want to show just `to_vec`, make it a separate statement, e.g.
```
# use risc0_zkvm::{
#   ExecutorEnv,
#   ExecutorEnvBuilder
# };
use risc0_zkvm::serde::to_vec;
```